chihacknight/decarbonize-my-state
# State detail user feedback

## Overview

Addresses feedback in #168:

* shortens height of state detail area chart
* shows percentage reduction on state detail top line instead of MMTCO2e
* updates data source citations with links to full citation on about page
* refactors terminology page as a list of definitions defined in `terminology-list.js` so we can use them throughout the site
* for terms used throughout the site, uses Bootstrap's `Tooltip` to hover/click for a definition popup without having to link off to the terminology page
* small text change for building _appliances_ on state detail

Closes #168 



### Demo

Shorter chart on state detail:

<img width="1497" alt="Screen Shot 2023-01-15 at 8 56 00 PM" src="https://user-images.githubusercontent.com/919583/212589186-491cdc79-c817-479b-83d6-d2b366ff8eb6.png">

Terminology tooltip hover:

<img width="1198" alt="Screen Shot 2023-01-16 at 12 28 10 AM" src="https://user-images.githubusercontent.com/919583/212612299-3fab6e04-7846-41e2-87d0-b664787d6101.png">



## Testing Instructions

- check the state detail page and confirm top line looks good and chart is smaller
- look for dashed underline words and confirm hovering over them shows a definition tooltip
- on mobile, confirm terminology tooltips work by tapping
